glean-core/examples/sample.rs

```diff
@@ -64,14 +65,16 @@ fn main() {
     let ping = ping_maker
         .collect_string(glean.storage(), glean.get_ping_by_name("core").unwrap())
         .unwrap();
-    println!("Ping:\n{}", ping);
+    println!("Core Ping:\n{}", ping);
 
-    let mut long_string = std::iter::repeat('x').take(49).collect::<String>();
+    // Cause some error reporting, which will land in the metrics ping.
+    // The `local_metric` itself is not in the metrics ping.
+    let mut long_string = std::iter::repeat('x').take(99).collect::<String>();
```

Comment:
I think there was a test case for this already, why should we clutter this sample a bit more?

Reply:
Hm, it was in there before and was useful for a quick one-off test.
But I'm also fine with keeping the sample much more minimal and thus removing this part.

Reply:
I personally feel like we should keep examples to the bare minimum :)